lib/hooks/useSwipeBlock/index.ts

```diff
@@ -3,15 +3,13 @@ import { useDrag } from "@use-gesture/react";
 import { useMediaQuery } from "usehooks-ts";
 
 interface SwipeBlockType {
-    expandable?: boolean;
```

Comment:
Reverting ActionSheet changes in this file from this [PR](https://github.com/deriv-com/quill-ui/pull/303/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519).